src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedContainer : NamedTypeSymbol
     {
+        internal const string StructFieldCaptureName = "<struct-field-capture-name>";
```

Comment:
I think you need to come up with a real name for this before we merge it. Changing baselines because the name changed is annoying and we don't want to do it more than we have to.